### PR TITLE
Add LOG_ERROR to default metrics_config value

### PR DESCRIPTION
**Pull request by jackmurdoch**

### CODE REVIEW

File: variables.tf [hunk 1]
```diff
@@ -258,7 +258,7 @@ variable "upload_token_lifetime_mins" {
   default = "1440"
 }
 variable "metrics_config" {
-  default = "{ \"CONTACT_UPLOAD\": 60, \"CHECK_IN\": 60, \"FORGET\": 60, \"TOKEN_RENEWAL\": 60, \"CALLBACK_OPTIN\": 60, \"DAILY_ACTIVE_TRACE\": 60, \"CONTACT_NOTIFICATION\": 60 }"
+  default = "{ \"CONTACT_UPLOAD\": 60, \"CHECK_IN\": 60, \"FORGET\": 60, \"TOKEN_RENEWAL\": 60, \"CALLBACK_OPTIN\": 60, \"DAILY_ACTIVE_TRACE\": 60, \"CONTACT_NOTIFICATION\": 60, \"LOG_ERROR\": 1 }"
```
Comment (jackmurdoch) [inline variables.tf]: @colmharte the rate limiting should be less than once per 60 seconds for this error I assume, but is once per second a sensible value?

Comment (colmharte) [reply variables.tf]: its once per 60 per person ? In which case thats ok

**Review by pmcgrath — APPROVED** [review]
lgtm

**Comment by pmcgrath** [comment]
@jackmurdoch We need to do our PRs targeting the **dev** branch
Will try to keep current matching upstream and do batch updates into that once a week or something like that

**I have chnaged the target branch already**

**Pull request merged**